Readme.md

```diff
@@ -1 +1,42 @@
-Cabal frontend
\ No newline at end of file
+#                                              cabal-phatom-frontend
+
+---
+
+## Technology
+
+---
+
+- Install NodeJS
+
+## Project setup
+
+---
+
+- Clone the repository
+- Install packages by running `npm install`
+- Run `npm run start` to start view in browser (for Development)
+- Run `npm run build` in production
+
+## DevDependencies
+
+---
+
+- babel-core,
+- babel-loader
+- babel-preset-env
+- babel-preset-react,
+- html-webpack-plugin
+- react-router-dom
+- react-hot-loader,
+- react-scripts
+- typescript
+- webpack@4.44.2,
+- webpack-cli
+- webpack-dev-server
```

Comment:
It's not necessary to have all these tools in README. We can find them in `package.json`